### ISSUE TITLE
Expandybird choking on replicatedservice-3.tgz

### ISSUE THREAD

**Issue by jackgr** [kubernetes/helm#485]
```
jackgr@jackgr-macbookpro:~/gopath/src/github.com/kubernetes/helm> helm deploy --name test1 gs://kubernetes-charts-testing/replicatedservice-v3.tgz
[ERROR] {"status":"Bad Request","message":"cannot expand configuration:expandybird response:\nerror expanding chart: test1: ExpandyBird cannot do this kind of expansion: %!!(MISSING)(EXTRA string=Expandybird)\n\u0026{[0xc2080824e0]}\n"}
```

/cc @sparkprime

**Comment by sparkprime** [comment]
We should probably just remove that check, as it allows deploying several versions of expandybird with different "names".  The expansion service should just assume that the thing it's given is of the kind it's designed to deal with.

**Comment by jackgr** [comment]
Sounds like a good approach. Probably best if you do it, since you know that code better than I do.

**Comment by sparkprime** [comment]
Delete this code
```
    if chartFile.Expander.Name != "ExpandyBird" {
        message := fmt.Sprintf("ExpandyBird cannot do this kind of expansion: ", chartFile.Expander.Name)
        return nil, fmt.Errorf("%s: %s", chartInv.Name, message)
    }
```

**Comment by sparkprime** [comment]
in cmd/expandybird/expander/expander.go

**Comment by jackgr** [comment]
Fixed by #486.
